Makefile

```diff
@@ -85,6 +85,10 @@ stop:
 	make stop-standalone
 	make stop-cluster
 
+restart:
+	make stop
+	make start
+
 test:
```

Comment:
It takes some time to run benchmarking. Can we move benchmarking to a separate target (e.g. `make benchmark`), so that we don't run them with `make test`?

Reply:
agree